tile_pages.py

```diff
@@ -161,9 +161,11 @@ def run(
         if '/Rotate' in self.in_doc.Root.Pages.keys():
             page_rot = self.in_doc.Root.Pages.Rotate
         
-        ref_p = next((p for p in self.page_range if p), None)
+        ref_p = self.page_range[0]
```

Comment:
The next() syntax is finding the first non-zero page.

Reply:
Ah, I see.